apps/help-with-fees/serviceaccount/ithc.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ${NAMESPACE}
+  namespace: ${NAMESPACE}
+  annotations:
+    azure.workload.identity/client-id: ""
```

Comment:
you probably need to have a value here if you are wanting to use that?

Reply:
@thomast1906 I'm in the process of finding the value for it.

Reply:
Sure - we will hold off merging until value added

Reply:
@thomast1906 ready to merge